interop/tapir/src/main/scala/caliban/interop/tapir/package.scala

```diff
@@ -162,17 +162,19 @@ package object tapir {
     override protected val additionalDirectives: List[__Directive] = Nil
   }
 
-  private def extractPath[I](input: EndpointInput[I]): String =
-    input
-      .asVectorOfBasicInputs(includeAuth = false)
-      .collect {
-        case EndpointInput.FixedPath(s, _, _) => s
+  private def extractPath[I](endpointName: Option[String], input: EndpointInput[I]): String =
+    endpointName.getOrElse(
```

Comment:
Maybe add some sanitization in case the name contains some spaces? Like turning `Add a book` into `AddABook`? In case people use actual sentences there.

Reply:
handled this part, pls have a look

as for `PathCapture` i'll try to do it

Reply:
By the way I’m fine if you prefer splitting this into 2 PRs and handle PathCapture later. 

Also, can you modify the docs in vuepress/docs/docs/interop.md to mention the new rule for naming fields?

Reply:
Yes I'd much rather do the second part as a separate PR

This somewhat depends on what we agree on in my other comment https://github.com/ghostdogpr/caliban/pull/375#discussion_r422998153

Reply:
Sounds good. Thanks for adding the docs.